homeassistant/components/velux/manifest.json

```diff
@@ -1,10 +1,18 @@
 {
   "domain": "velux",
   "name": "Velux",
-  "codeowners": ["@Julius2342", "@DeerMaximum"],
+  "codeowners": ["@Julius2342", "@DeerMaximum", "@pawlizio"],
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/velux",
+  "integration_type": "hub",
   "iot_class": "local_polling",
+  "issue_tracker": "https://github.com/Julius2342/pyvlx/issues",
   "loggers": ["pyvlx"],
-  "requirements": ["pyvlx==0.2.21"]
+  "requirements": ["pyvlx==0.2.23"],
```

Comment:
You are supposed to update the dependency in a separate PR.

Reply:
PR #115483 created